dnsext-dnssec/DNS/SEC/Verify/Verify.hs

```diff
@@ -126,10 +126,12 @@ verifyRRSIGwith RRSIGImpl{..} now dnskey@RD_DNSKEY{..} rrsig@RD_RRSIG{..} rrset_
 
 {- Canonical RR Ordering within an RRset
    https://datatracker.ietf.org/doc/html/rfc4034#section-6.3
-   Assumes same ( rrname, rrtype, rrclass, rrttl ).
-   Same order between RData and RR, under same ( rrname, rrtype, rrclass, rrttl ). -}
+   "RRs with the same owner name,
```

Comment:
Would you add "RFC 4034 Section 6.3:"

Reply:
https://github.com/kazu-yamamoto/dnsext/pull/129/commits/edb8a00258d42aa0aca94e82f52fb02ee1fa793a